sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -13,11 +13,17 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
 
         self._user = new Sourcemash.Models.User();
         self._feeds = new Sourcemash.Collections.Feeds([], {allFeeds: true});
-        self._categories = new Sourcemash.Collections.Categories();
+        self._categories = new Sourcemash.Collections.Categories([], {allCategories: true});
```

Comment:
See the HTML. It literally works identically to how you wrote feeds in the UI redesign